core/src/main/java/com/netflix/conductor/core/events/EventProcessor.java

```diff
@@ -132,11 +131,14 @@ public void refresh() {
 		});
 	}
 
-	private String handleEventBus(String event) {
-		if (StringUtils.isEmpty(eventBus)) {
-			return event;
-		}
-		return event.replace(EVENT_BUS_VAR, eventBus);
+	private List<EventHandler> getEventHandlersForEvent(String event) {
+		return ms.getEventHandlers().stream()
+			.filter(EventHandler::isActive)
+			.filter(e -> {
+				String replaced = (String) pu.replace(e.getEvent());
+				return replaced.equalsIgnoreCase(event);
+			})
+			.collect(Collectors.toList());
 	}
```

Comment:
Main changes mostly here, in `getEventHandlersForEvent` method. It uses the `ParametersUtils` class to handle all possible variables in the event name (e.g. Event Bus or event/queue name) from the `${..}` variable)